TiKiTuTo/Model/BusinessLogic/GameLogic/GameLogicMatch.cs

```diff
@@ -10,20 +10,20 @@ public class GameLogicMatch
         //GameLogicRound GameLogicRound { get; set; }
         InputHandler InputHandler { get; set; }
         JSONService JSONService { get; set; }
-
+        bool isTimerFinished = false;
```

Comment:
Do I understand correctly that this is a property so that both RunMatch and StartMatchTimer have access to its value?